src/createQueryGroup.ts

```diff
@@ -23,6 +23,8 @@ export type QueryGroupOptions = {
   onDispose?: () => void,
 }
 
+const createQueryId = createSequence()
```

Comment:
most of these global identifiers now were only used inside of the specific "getXKey" services, so I decided to move them all out of `./createSequence`